app/__main__.py

```diff
@@ -124,7 +125,12 @@ async def on_message(message: discord.Message) -> None:
 @app_commands.describe(message="The message to send")
 async def send(interaction: Interaction, message: str) -> None:
     """Send a message to the current channel."""
-    await interaction.response.send_message(message)
+    if interaction.channel not in interaction.client.database.get_enabled_channels(interaction.guild):
```

Comment:
database.get_enabled_channels has been replaced with database.get_channels I believe and also needs to be awaited